### ISSUE TITLE
Test failed Feature Flags contract

### ISSUE THREAD

**Issue by lordbah** [dashpay/platform#2222]
## Expected Behavior

Tests should pass.

## Current Behavior

✖ @dashevo/feature-flags-contract at packages/feature-flags-contract 4.891s
  ✖ test 4.881s
    ✖ test:unit 4.864s
      ✖ $ mocha 'test/unit/**/*.spec.js' 4.779s
        │ 
        │ 
        │   Feature Flags contract
        │     1) "before each" hook for "should have a valid contract definition"
        │ 
        │ 
        │   0 passing (2s)
        │   1 failing
        │ 
        │   1) Feature Flags contract
        │        "before each" hook for "should have a valid contract definition":
        │      JsonSchemaError: "position" is a required property, path: /properties/block/properties/maxBytes

## Steps to Reproduce (for bugs)

1. yarn test
2.
3.
4.

## Your Environment

repo cloned as of 5e996edd879222d51b9843a4625a04db4c1b2daf (7-Oct-2024)
Built per README
PopOS 22.04 (Ubuntu)

**Comment by pshenmic** [comment]
Yeah, feature flags data contract schema is invalid and missing `position`, I don't know why, most likely has to be forgotten to update. I'm not sure we use this data contract anymore though

**Comment by shumkov** [comment]
We don't use this data contract anymore. We should remove it form repo
